src/ensembl/src/content/app/help/components/labelled-app-icon/LabelledAppIcon.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import upperFirst from 'lodash/upperFirst';
+
+import { ReactComponent as SpeciesSelectorIcon } from 'static/img/launchbar/species-selector.svg';
+import { ReactComponent as BrowserIcon } from 'static/img/launchbar/browser.svg';
+import { ReactComponent as EntityViewerIcon } from 'static/img/launchbar/entity-viewer.svg';
+
+import styles from './LabelledAppIcon.scss';
+
+type AppName = 'speciesSelector' | 'genomeBrowser' | 'entityViewer';
+
+type Size = 'regular' | 'small';
+
+type Props = {
+  app: AppName;
+  size: Size;
+};
+
+const appNameToComponent: Record<AppName, React.FunctionComponent> = {
```

Comment:
would it be better to say `appNameToIconComponent`?

Reply:
changed to `appNameToIcon`